main.nf

```diff
@@ -431,6 +442,31 @@ if(params.aligner == 'hisat2' && !params.hisat2_index && params.fasta){
         """
     }
 }
+
+
+/*
+ * PREPROCESSING - Create Salmon transcriptome index
+ */
+if(params.transcriptome){
+  process salmon_index {
+      tag "$transcriptome.simpleName"
+      publishDir path: { "${params.outdir}/reference_transcriptome" },
+                 mode: 'copy'
+
+      input:
+      file transcriptome from tx_fasta_ch
+
+      output:
+      file 'salmon_index' into index_ch
+
+      script:
+      """
+      salmon index --threads $task.cpus -t $transcriptome -i salmon_index
```

Comment:
salmon is not in the container
![image](https://user-images.githubusercontent.com/34273025/58952665-041aef00-879d-11e9-95e7-610ce0298719.png)

So I get the following error
![image](https://user-images.githubusercontent.com/34273025/58954841-f49ea480-87a2-11e9-9451-7ecb3e6bb35c.png)

in my repo I am using
`  - bioconda::salmon=0.13.1`

Reply:
thanks, we are waiting for the other #210 to have the container with salmon,